pkg/kn/commands/completion_helper_test.go

```diff
@@ -72,15 +75,50 @@ var (
 	testNsServices = []v12.Service{testSvc1, testSvc2, testSvc3}
 
 	fakeServing = &servingv1fake.FakeServingV1{Fake: &clienttesting.Fake{}}
-	knParams    = &KnParams{
+)
+
+var (
+	testBroker1 = v13.Broker{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Broker",
+			APIVersion: "eventing.knative.dev/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{Name: "test-broker-1", Namespace: testNs},
+	}
+	testBroker2 = v13.Broker{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Broker",
+			APIVersion: "eventing.knative.dev/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{Name: "test-broker-2", Namespace: testNs},
+	}
+	testBroker3 = v13.Broker{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Broker",
+			APIVersion: "eventing.knative.dev/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{Name: "test-broker-3", Namespace: testNs},
+	}
+	testNsBrokers = []v13.Broker{testBroker1, testBroker2, testBroker3}
+
+	fakeEventing = &fake.FakeEventingV1{Fake: &clienttesting.Fake{}}
+)
+
+var knParams = initialiseKnParams()
```

Comment:
Might be part of `var()` block above?

Reply:
I thought it would be neater to create diff var blocks for different resources (the one above it is for service resource)

Reply:
OK, lets keep it then! :)